core/src/navigation/ContextSwitcher.html

```diff
@@ -49,10 +49,14 @@
               on:click="{() => goToOption(node, selectedOption)}"
               data-testid="{getTestId(node)}"
             >
+              {#if customOptionsRenderer}
+              {@html customOptionsRenderer(node)}
+              {:else}
               <a
                 class="fd-menu__item {node === selectedLabel ? 'is-selected' : ''}"
```

Comment:
How is the "selected" reflected for custom renderers? Guess we should add that `selectedLabel` as a parameter for `customOptionsRenderer`.

Reply:
👍 good catch!